docs/API/Elements.md

```diff
@@ -5,18 +5,60 @@ title: Elements
 
 The Elements API is a powerful abstraction that allows users to manipulate any property on any element through a handful of simple methods. This is important because these few methods allow for easy networking or persistent state. 
 
+To change these properties you will need to reference an object. The most common form to reference an object would be:
+
+```js
+var a = this; 
+```
+You can then use this reference to change other properties.
+```js
+var a = this; 
+a.visible = true; // set object visible to on
+```
+
+### Common properties
+
+Many functions use ID's.
+
+```js
+var a = this.id;   // Read-only. Gets the element's ID
+```
+Each element has a type, currently there are a few different object types, <i>LightWidget, ContentWidget, KinectWidget, ButtonWidget, and TextWidget </i>. Any newly created asset will be a ContentWidget by default.
+
+```js
+var a = this.type; //Read-only. Get's element type. 
+```
+
+<code>This</ code> is a helpful keyword you can use to return most information you will need about your current object.
```

Comment:
Looks like you've got an extra space in the closing tag `</ code>` that I think you need to remove for the markdown to read properly